gm4_blossoming_pots/translations.csv

```diff
@@ -0,0 +1,3 @@
+key,en_us
+text.gm4.guidebook.module_desc.blossoming_pots,Greatly expands the selection of potable plants. Use Decorated Pots as just abnormally large flower pots. Decorate in Style!
+text.gm4.guidebook.blossoming_pots.description,"Interacting with a Decorated Pot or empty Flower Pot while holding any one of the supported items will plant it in the pot.\n\nCheck the full supported item list on the wiki."
```

Comment:
As catter said in another comment, this should have consistent capitalization as the rest of the instances of this text.